public/download.php

```diff
@@ -9,64 +14,71 @@
 <div id="mainpage">
     <div id='fullcontainer'>
 
-        <h2>Downloads</h2>
-
-        <p class="embedded mb-5">
-            <b>Upgrade:</b> If you were brought to this page because the emulator told you
-            that a new version is available, download the new version and
-            extract it over the existing folder. This way you won't lose
-            any save files that you may have.
-        </p>
+        <h2>Emulators supporting RetroAchievements</h2>
```

Comment:
```suggestion
        <h2 class="mb-6">Emulators supporting RetroAchievements</h2>
```

nit: My eyes immediately jumped to how close the H1 is to the first emulator item in the list. I think this is due to the two elements having the same font size. This change is subtle, but gives just a pinch of breathing room.

**BEFORE**
![Screenshot 2023-04-15 at 5 56 31 PM](https://user-images.githubusercontent.com/3984985/232254844-1cedf77f-e3f0-4102-8927-7e638be665aa.png)

**AFTER**
![Screenshot 2023-04-15 at 5 56 22 PM](https://user-images.githubusercontent.com/3984985/232254851-115b64e2-4e59-49e9-b2ef-22da1ebe276d.png)